File: NetBeans_IDE/JsonPL/src/com/middlemind/JsonPL/ProcessUrlFindRunner.java
package com.middlemind.JsonPL;

/**
 *
 * @author brusc
 */
public class ProcessUrlFindRunner implements Runnable {
    private final JsonPlState main;
    private final String url;
    
    public ProcessUrlFindRunner(JsonPlState Main, String Url) {
        main = Main;
        url = Url;
    }
    
    @Override
    public void run() {
        main.processUrlFindExec(url, true);
    }    
}

File: storage/alt_interpreters/VisualStudio_IDE/JsonPL/JsonPL/com/middlemind/JsonPL/ProcessUrlFindRunner.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace com.middlemind.JsonPL
{
    public class ProcessUrlFindRunner
    {
        private JsonPlState main;
        private string url;

        public ProcessUrlFindRunner(JsonPlState Main, string Url)
        {
            main = Main;
            url = Url;
        }

        public void run()
        {
            main.processUrlFindExec(url, true);
        }
    }
}
